### DISCUSSION TITLE
http://pism.github.io/ is 404 for now

### DISCUSSION THREAD

**Discussion by bueler** [pism/pism#470] [Ideas]
The PISM docs are are http://pism.github.io/pism/, and the "old" website is at pism-docs.org.  However, for now http://pism.github.io/ gives 404.

Can/should something go there?  Redirect to pism-docs?  Or move the dokuwiki content at pism-docs.org to pism.github.io?

[Trying out discussions ...]

**Comment by ckhroulev** [comment]
Right now http://pism.github.io/ points to the new PISM website (same as http://pism.io).

The old website at http://pism-docs.org should go away as soon as I release the "stable" PISM version.
